packages/gatsby-plugin-gatsby-cloud/src/index.js

```diff
@@ -0,0 +1 @@
+// noop
```

Comment:
This file can be deleted as we have it in the root already

Reply:
Done!